src/test/java/JavaClass1/JavaClassWork.java

```diff
@@ -23,7 +23,10 @@ public class JavaClassWork {
     *376783486593
     *creating second remote branch
     *
-    *
+    *29348727y9397832792
```

Comment:
No need to create new  remote Branch every time with small new changes, especially when 1st PR is still under review. 
So please push these changes to SamRemoteCommit only